contrib/curry/ingress-external/kustomization.yaml

```
@@ -10,7 +10,7 @@ replacements:
     targets:
       - select:
           kind: Ingress
-          labelSelector: app.kubernetes.io/managed-by=kustomize-ingress
+          labelSelector: app.kubernetes.io/managed-by=kustomize, app.kubernetes.io/instance=external
```

Just a NIT (I am going to write only once) but I think that this way is more readable.
```suggestion
          labelSelector: >-
            app.kubernetes.io/managed-by=kustomize,
            app.kubernetes.io/instance=external
```

---

LGTM, seems more verbose this way. But I'll do this on a different PR, I don't want to rebase a lot of changes 😅